2025-04-14-January model to send for review.nlogo

```diff
@@ -0,0 +1,1081 @@
+globals
+[
```

Comment:
I wound up commenting for myself:

```
  ;; advantage-B <-- epsilon
  ;; trial-size <-- _n_
  ;; number_turtles <-- _K_
  ;; policymaker_connectedness <-- _k_

  ;; pg-B <-- _p_sub_B_
  
  ``` 

I recommend using variables from the paper for naming your Netlogo controls, or adding labels into the UI with the matching paper formal variables. It can be very confusing to a reviewer having a large difference between control names and system variables.